packages/widget/src/components/widget-app/widget-app.ts

```diff
@@ -176,63 +196,65 @@ class ConnectDialog extends LitElement {
-    } else {
-      return html`<div>
-        <p>EVM Account: ${this.walletManager.accountData}</p>
-        <p>Network: ${this.chainId}</p>
-        <p>SDK Status: ${this.sdkManager?.status}</p>
-        ${this.sdkManager?.status === 'idle'
-          ? html`<button @click=${this.initSdk}>initialize sdk</button>`
-          : undefined}
-        ${when(
-          this.sdkManager?.status !== 'idle' && this.domains && this.homechain,
-          () => html`
-            <network-selector
+    return html`
+      <section class="widgetContainer">
+        <form @submit=${this.handleTransfer}>
+          <section class="switchNetwork">
+            <span>${switchNetworkIcon}</span>
+            <span>Switch Network</span>
+          </section>
+          <section>
+          <network-selector
               .directionLabel=${'from'}
               .networkIcons=${true}
               .homechain=${this.homechain}
-              isHomechain=${true}
+              .isHomechain=${true}
               .selectedNetworkChainId=${this.chainId}
               .disabled=${true}
             ></network-selector>
-            <network-selector
+          </section>
+          <section>
+          <network-selector
               .domains=${this.destinationDomains}
               .directionLabel=${'to'}
               .networkIcons=${true}
               .selectedNetworkChainId=${this.selectedNetworkChainId}
             ></network-selector>
-            <amount-selector
+            </section>
+          <section>
+          <amount-selector
               .disabled=${false}
               .selectedNetworkChainId=${this.selectedNetworkChainId}
               .resources=${this.resources}
             >
             </amount-selector>
-            ${choose(this.sdkManager?.status, [
-              [
-                'initialized',
-                () =>
-                  html`<button @click=${this.createTransfer}>
-                    Create transfer
-                  </button>`
-              ],
-              [
-                'transferCreated',
-                () =>
-                  html`<button @click=${this.approveTokens}>Approve</button>`
-              ],
-              [
-                'approvalsCompleted',
-                () =>
-                  html`<button @click=${this.performDeposit}>Transfer</button>`
-              ]
-            ])}
-          `
-        )}
-      </div>`;
-    }
+          </section>
+          <section>
+            Transfer to the same address
+          </section>
+          <section>
+            ${when(
+              !this.walletManager || !this.walletManager.accountData,
+              () =>
+                html`<button
+                  @click=${this.connect}
+                  type="button"
+                  class="actionButton"
+                >
+                  Connect
+                </button> `,
+              () =>
+                html`<button type="submit" class="actionButtonReady">
```

Comment:
`type="submit"` forces the page to refresh when the button is pushed. Should rather use `type="button"` and handle the form submission in the `onClick` handler.